src/client/pages/Varen/useVarenThemaData.hook.ts

```diff
@@ -51,12 +25,11 @@ export function useVarenThemaData() {
   );
 
   return {
-    gegevensAanvrager,
     varenRederRegistratie,
     tableConfig,
     isLoading: isLoading(VAREN),
     isError: isError(VAREN),
     varenVergunningen,
-    linkListItems,
+    linkListItems: varenMeerInformatieLink,
```

Comment:
Dit is denk ik niet de bedoeling. `linkListItems` is overal `LinkProps[]` waarom zou je er hier iets anders mee doen?